src/composables/config/useDriver.ts

```diff
@@ -123,12 +124,29 @@ export const dataStatistics = () => {
       .then((res: any) => {
         const statistics: string = res?.data || ''
         nodeStatisticData.value = statistics.replace(/(#)(.*)(\n)/g, '')
+        setTimer(type, params)
```

Comment:
I have faced a situation where the interface was still being requested when the component was destroyed, so the timer was not on and therefore not be cleared, and the request was completed after the component was destroyed and the timer was on, so perhaps you could try this to see if there is a hidden problem.